<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionWindowClass</class>
 <widget class="QWidget" name="QuestionWindowClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Franklin Gothic Book</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Question</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame {
	border-radius: 20px;
}

QWidget {
	background-color: #392615;
}</string>
  </property>
  <widget class="QLabel" name="banner">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>-11</y>
     <width>670</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource/Triviador.qrc">:/Banner/question/EmptyBanner.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="question">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>4</y>
     <width>620</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
background: transparent;</string>
   </property>
   <property name="text">
    <string>Care este capitala României?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="timeProgressBar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>400</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">.QProgressBar {
	background-color: #000000;
	border-style: solid;
	border-radius: 7px;
}

QProgressBar::chunk {
	background-color: #B73131;
	border-radius: 7px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="questionTypes">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>130</y>
     <width>661</width>
     <height>350</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="multiple">
    <widget class="QPushButton" name="answer1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
background-color: rgb(83, 66, 50);
checked: {background-color: rgb(255, 244, 160)};
</string>
     </property>
     <property name="text">
      <string>București</string>
     </property>
    </widget>
    <widget class="QPushButton" name="answer2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">/*background-image: url(:/Others/question/Plank.svg);*/
/*border: none;*/
color: #ffffff;
background-color: rgb(83, 66, 50);
checked: {background-color: rgb(255, 255, 127)}</string>
     </property>
     <property name="text">
      <string>Paris</string>
     </property>
    </widget>
    <widget class="QPushButton" name="answer3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">/*background-image: url(:/Others/question/Plank.svg);*/
/*border: none;*/
color: #ffffff;
background-color: rgb(83, 66, 50);
checked: {background-color: rgb(255, 255, 127)}</string>
     </property>
     <property name="text">
      <string>Londra</string>
     </property>
    </widget>
    <widget class="QPushButton" name="answer4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">/*background-image: url(:/Others/question/Plank.svg);*/
/*border: none;*/
color: #ffffff;
background-color: rgb(83, 66, 50);
checked: {background-color: rgb(255, 255, 127)}</string>
     </property>
     <property name="text">
      <string>Dubai</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hammerButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #906B48;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/Triviador.qrc">
       <normaloff>:/Others/question/Hammer.svg</normaloff>:/Others/question/Hammer.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="numerical">
    <widget class="QLabel" name="plank">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource/Triviador.qrc">:/Others/question/Plank.svg</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="answerInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: #725C48;
color: #ffffff;
padding: 5px;</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #5F9E2D;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/Triviador.qrc">
       <normaloff>:/Others/question/PlayArrow.svg</normaloff>:/Others/question/PlayArrow.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="telescopeButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #906B48;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/Triviador.qrc">
       <normaloff>:/Others/question/Telescope.svg</normaloff>:/Others/question/Telescope.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="parrotButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #906B48;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/Triviador.qrc">
       <normaloff>:/Others/question/Parrot.svg</normaloff>:/Others/question/Parrot.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="flag1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
padding: 10px;
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="flag3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>160</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
padding: 10px;
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="flag2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>160</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
padding: 10px;
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="flag4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>160</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
padding: 10px;
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/Triviador.qrc"/>
 </resources>
 <connections/>
</ui>
